components/build-service/production/base/manager_resources_patch.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: build-service-controller-manager
+  namespace: build-service
+spec:
+  template:
+    spec:
+      containers:
+      - name: manager
+        resources:
+          limits:
+            cpu: 500m
+            memory: 2Gi
+          requests:
+            cpu: 250m
+            memory: 768Mi
```

Comment:
if you want to prevent eviction, you must set memory requests=limits. If the memory usage is above requests, e.g. >=768Mi and the node is running low on memory(pod scheduling is done based on requests value, not limit), the pod will be considered for eviction.

That said. it the current proposed memory request is above typical usage, we can leave the change as is.

Reply:
I looked at current usage and gave slightly higher value than typical, this should prevent eviction. I still don't want to consume more than actually needed for now.